### PR TITLE
chore: update readme

### PR DESCRIPTION
**Pull request by jugaltheshah**
Add default Rabbit url to readme as if you previously installed desk, it has the wrong default.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,8 @@ This will build new images for the RabbitMQ, MongoDB and Redis services and then
 
 And that's it. Congratulations on getting your local env ready for some developing.
 
+Please note that when you run a service locally, RABBIT_URL should be "amqp://localhost/db".
```
Comment (cilindrox) [inline README.md]:
```suggestion
Please note that when you run a service locally, RABBIT_URL should be "amqp://localhost:5672/db".
```

Also what about just adding a `RABBIT_URL=amqp://rabbit:5672/db` to the `.env` instead of this?

Comment (jugaltheshah) [reply README.md]: Good thought. I'm not sure about most people though, but I personally tend to have this project always running and I open and close (and source .desk files) the services intermittently. If you source a desk file with `RABBIT_URL=amqp://localhost` after this, it will still break.

Tl;dr: how about both?

Comment (cilindrox) [reply README.md]: sounds good. Careful with keeping these running: _anyone in the LAN_ can now connect to your machine and be nasty if you don't have a firewall properly configured.
